src/components/Speciality/books.js

```diff
@@ -44,66 +44,59 @@ const BooksBox = styled.a`
   `}
 `
 
-const BooksSection = ({ speciality }) =>
-  speciality.externalResources.filter(
-    additionalInfo => additionalInfo.type === `Book`
-  ).length ? (
+const BooksSection = ({ books, title }) =>
+  books.length ? (
```

Comment:
probably add a `books && books.length` just to make sure it won't break?

Reply:
Changed the getExternalType to:
```
  const getExternalType = type =>
    speciality.externalResources.filter(
      additionalInfo => additionalInfo.type === type
    ) || []
```